lib/SPIRV/SPIRVToOCL.h

```diff
@@ -182,6 +182,9 @@ class SPIRVToOCLBase : public InstVisitor<SPIRVToOCLBase> {
   /// using separate maps for OpenCL 1.2 and OpenCL 2.0
   virtual Instruction *mutateAtomicName(CallInst *CI, Op OC) = 0;
 
+  // Transform FP atomic opcode to corresponding OpenCL function name
+  virtual std::string FPAtomicToOCLBuiltinName(Op OC) = 0;
```

Comment:
In order to fix `clang-tidy` issue you can rename this function into something like `mapFPAtomicName` to be closer to existing `mapAtomicName` which you modify here

Reply:
Done, thanks for the suggestion!